Update usage

nfebe opened the pull request:
Improve script usage and add test scope.

kfogel commented on README.md:
```diff
@@ -0,0 +1,11 @@
+# permanent-rclone-qa
+
+This project is intended for testing the [SFTP service](https://github.com/PermanentOrg/sftp-service). This repository 
```
Let's refer back to the Zulip conversation from here, as we [discussed](https://chat.opentechstrategies.com/#narrow/stream/73-Permanent/topic/QA/near/155549), so that the reader can go get more context if they need it.

mithuna replied:
okay. Will add that too.

kfogel commented on README.md:
```diff
@@ -0,0 +1,11 @@
+
+## License
+
+This code is free software licensed as [CC0v1.0](LICENSE), or at your
```
There's not just code here, there will be data as well, so say "This code and data is licensed under ...".

The part about using any later version of the license can go away.  That language comes from the Free Software Foundation's General Public License (the GPL), and this is a Creative Commons license -- it doesn't come from the FSF at all.

slifty commented on README.md:
```diff
@@ -0,0 +1,10 @@
+
+This project is intended for testing the [SFTP service](https://github.com/PermanentOrg/sftp-service). This repository 
+will have a bunch of helper tools in a separate repository, whose purpose is just to help us do manual QA, and then 
+later that gets turned into an automated tests in the SFTP service repository. Refer our discussions 
```
> Refer our discussions

should be

> Refer to our discussions

slifty replied:
"into an automated tests" should be "into automated tests"

mithuna replied:
Thank you.

slifty commented on README.md:
```diff
@@ -0,0 +1,10 @@
+# permanent-rclone-qa
```
Let's make this title something more human readable / descriptive (e.g. `Permanent rclone QA Resources`)

slifty reviewed (changes requested):
Two small typos and a suggestion!

mithuna commented:
Made changes as per requested. Please re-review @slifty @kfogel when you can.

slifty reviewed (approved):
Thanks!  My comments are addressed

mithuna commented:
@**Karl Fogel**  Do you have any comments on this PR? or can I merge this?

kfogel reviewed (approved):
Let's merge.  We can add more to the README later.

nfebe committed:
Improve usage of upload-test script

This commit makes it possible to:

- Specify archive path when calling script
- Use any remote name not limited to permanent-dev or prod
- Includes the requirements for the apod downloader in requirements.txt and
 updates the readme with the correct run instructions.

Signed-off-by: Fon E. Noel NFEBE <fenn25.fn@gmail.com>

nfebe committed:
Add QA scope

Document what is currently covered and what is not.

Signed-off-by: Fon E. Noel NFEBE <fenn25.fn@gmail.com>

jvasile commented on README.md:
```diff
@@ -12,20 +12,32 @@ discussion](https://chat.opentechstrategies.com/#narrow/stream/73-Permanent/topi
+
+1. `python -m venv venv`
+2. `source venv/bin/activate`
+3. `pip install requirements.txt`
```
Don't you need `pip install -r requirements.txt`

jvasile commented on README.md:
```diff
@@ -12,20 +12,32 @@ discussion](https://chat.opentechstrategies.com/#narrow/stream/73-Permanent/topi
 
+**NB: The `--archive-path` argument specifies the route to the the specific permanent archive to which uploads would be made. So how do you get the archive path? See [Constructing archive path](https://github.com/permanentOrg/sftp-service/#downloading-from-permanent)**
+
+*That said, the archive path used in the sample command would have to updated to match some archive created on Permanent.org*
```
s/to updated/to be updated/
